zerver/actions/create_realm.py

```diff
@@ -279,7 +292,16 @@ def do_create_realm(
         )
 
         create_system_user_groups_for_realm(realm)
-        set_default_for_realm_permission_group_settings(realm)
+
+        group_settings_defaults_for_org_types = {
+            "can_create_public_channel_group": {
+                Realm.ORG_TYPES["education_nonprofit"]["id"]: SystemGroups.ADMINISTRATORS,
+                Realm.ORG_TYPES["education"]["id"]: SystemGroups.ADMINISTRATORS,
+            }
+        }
```

Comment:
Used this approach so that we need to fetch system user groups only once and the code to set the defaults can be inside a single function.

Reply:
Yeah, that seems fairly clean.